### ISSUE TITLE
[NOT important] Complx Screen Scrolling with Menu

### ISSUE THREAD

**Issue by jimjang95** [TricksterGuy/complx#115]
**Describe the bug**
Honestly not much of a game breaking bug but for some reason if you click one of the menu options on top and "scroll" through the menu options, the main screen scrolls up, too.

Works with both keyboard and mouse.

**To Reproduce**
1. Click any of the menu items. (File, View, State, Debug, Test, Help)
2. Move your mouse to another menu item. / Use your keyboard to move to another menu item.
 Bug appears when the selected item is a:
    - Menu Category (File, View, State, Debug, Test, Help)
    - Sub options with sub options ( View >> Hide Addresses, State >> Control ...)
        : In the case of sub options, screen scrolls both when the sub option is loaded + when once moves into the sub option with the Keyboard Right key. (left key / mouse has no effect on moving into sub options)

**Screenshots**
![bugreport](https://user-images.githubusercontent.com/11640155/46695454-8bce4f00-cc4a-11e8-8420-010e857fbb19.gif)

**Comment by ausbin** [comment]
fwiw i can't reproduce this in GNOME on Debian Stretch

However when I alt-tab to another window and back, complx does scroll down one entry

**Comment by TricksterGuy** [comment]
Yeah well aware of this as far as attempting to fix it not a priority right now.
